src/components/utils/Markdown.tsx

```diff
@@ -12,19 +13,31 @@ export default class Markdown extends React.Component<MarkdownProps> {
 
   constructor(props) {
     super(props);
-    this.renderer = new HtmlRenderer({ safe: true });
+    this.renderer = new HtmlRenderer();
     this.parser = new Parser();
   }
+
   shouldComponentUpdate(nextProps) {
-    return this.props.text !== nextProps.text;
+    return (
+      this.props.text !== nextProps.text || this.props.termToHighlight !== nextProps.termToHighlight
+    );
+  }
+
+  highlightTermInMarkdown(content: string, term: string) {
+    if (!term) {
+      return content;
+    }
+
+    return content.replace(new RegExp(term, 'gi'), match => `<mark>${match}</mark>`);
```

Comment:
It doesn't use `this` please extract it as a normal function.

Reply:
Done.